imageio/plugins/ffmpeg.py

```diff
@@ -293,8 +295,8 @@ def _open(
             regex_match = re.match(r"<video(\d+)>", self.request.filename)
             if regex_match:
                 self.request._video = self.request.filename
-            # Specify input framerate?
-            if self.request._video:
+            # Specify input framerate? (only on macOS)
+            if self.request._video and platform.system().lower() == "darwin":
                 if "-framerate" not in str(self._arg_input_params):
                     self._arg_input_params.extend(["-framerate", str(float(fps or 30))])
```

Comment:
Let's set the framerate to 15 in this case, which seems better supported on MacOS. See  #525 and the snippet you posted.

Reply:
Sounds good. I'll get right on it.